style.css

```diff
@@ -1,7 +1,89 @@
 html {
-  font-size: 10px;
+  font-size: 12px;
 }
```

Comment:
I don't see the need for this since each section has its own font-size. Kindly remove it.